parser/parser_test.go

```diff
@@ -45,29 +46,6 @@ func TestLetStatements(t *testing.T) {
 	}
 }
 
-func TestLetStatements_InvalidIdentifiers(t *testing.T) {
```

Comment:
obsolete because now we have `p.noPrefixParseFnError()` method, which means that along with the usual error, we also have extra "prefix not found" errors.